src/index.html

```diff
@@ -11,12 +11,111 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
+
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#scr/images">
+        <img
+          src="https://s3-alpha-sig.figma.com/img/833f/9aab/47d660efd384318772b6b8040db831aa?Expires=1728864000&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=l4p413eY9BSgqjAH-YHSf2j-OH0qW82oS-P0vNIIFqEbwdj34ZzpB3yQ8tZK2NoYca6fgB7j~-fAm0EFdGvNDkzWhsvTOUDlJ-Kfex7derqR90AWI9ekULcpdP73PtCkZoCU-eVyfOGzTKpmoCnk~qpRQTcuDFfhxQm~hoYpMagcYY1mN5-gvT-7R-cSvBZZeCsahEX-UJoC0txzt3aYJDJtCOZVQX~UEfOPREmaOEaqZ-RRYLCPI~cj64eT15rK-JDtFLtcz7PLu9CcT8aQQpSCeUO~JH-LwAwtOucqRT0dbx70av0fYYebqqRUGTQf-X~SyOgLhPKT3iPu249Jbw__"
+          alt="logo"
```

Comment:
The `alt` attribute should describe the image content, not just say 'logo'. It should be more descriptive about what the image represents.